shell/src/main/java/tachyon/shell/TfsShell.java

```diff
@@ -757,171 +126,17 @@ public void printUsage() {
-   *
-   * @param argv [] Array of arguments given by the user's input from the terminal
-   * @throws IOException
-   */
-  public void rename(String[] argv) throws IOException {
-    TachyonURI srcPath = new TachyonURI(argv[1]);
-    TachyonURI dstPath = new TachyonURI(argv[2]);
-    try {
-      TachyonFile fd = mTfs.open(srcPath);
-      if (mTfs.rename(fd, dstPath)) {
-        System.out.println("Renamed " + srcPath + " to " + dstPath);
-      } else {
-        throw new IOException("mv: Failed to rename " + srcPath + " to " + dstPath);
-      }
-    } catch (TachyonException e) {
-      throw new IOException(e.getMessage());
-    }
-  }
-
-  public void report(TachyonURI path) throws IOException {
-    try {
-      TachyonLineageFileSystem.get().reportLostFile(path);
-      System.out.println(path + " has reported been reported as lost.");
-      listLineages();
-    } catch (TachyonException e) {
-      throw new IOException(e.getMessage());
-    }
-  }
-
-  /**
-   * Removes the file specified by argv.
-   *
-   * @param path The TachyonURI path as the input of the command
-   * @throws IOException
-   */
-  public void rm(TachyonURI path) throws IOException {
-    TachyonFile fd;
-    FileInfo fInfo;
-    try {
-      fd = mTfs.open(path);
-      fInfo = mTfs.getInfo(fd);
-    } catch (TachyonException e) {
-      throw new IOException(e.getMessage());
-    }
-
-    if (fInfo.isFolder) {
-      throw new IOException("rm: cannot remove a directory, please try rmr <path>");
-    }
-
-    try {
-      mTfs.delete(fd);
-      System.out.println(path + " has been removed");
-    } catch (TachyonException e) {
-      throw new IOException(e.getMessage());
-    }
-  }
-
-  /**
-   * Removes the file or directory specified by argv. Will remove all files and directories in the
-   * directory if a directory is specified.
+   * Take out the command name arg
    *
-   * @param path The TachyonURI path as the input of the command
+   * @param argv the input args
+   * @return args without the command name
    */
-  public void rmr(TachyonURI path) throws IOException {
-    try {
-      DeleteOptions options = new DeleteOptions.Builder().setRecursive(true).build();
-      TachyonFile fd = mTfs.open(path);
-      mTfs.delete(fd, options);
-      System.out.println(path + " has been removed");
-    } catch (TachyonException e) {
-      throw new IOException(e.getMessage());
+  private static String[] stripNameArg(String[] argv) {
+    String[] args = new String[argv.length - 1];
+    for (int i = 1; i < argv.length; ++ i) {
+      args[i - 1] = argv[i];
     }
-  }
-
-  /**
-   * Displays the size of a file or a directory specified by argv.
-   *
-   * @param path The TachyonURI path as the input of the command
-   */
-  public void du(TachyonURI path) throws IOException {
-    long sizeInBytes = getFileOrFolderSize(mTfs, path);
-    System.out.println(path + " is " + sizeInBytes + " bytes");
+    return args;
```

Comment:
This might not be worth being a helper function since it's only called once and the name is somewhat confusing.